libraries/redcore/layouts/searchtools/default.php

```diff
@@ -23,7 +23,7 @@
 	'orderFieldSelector'  => '#list_fullordering'
 );
 
-$data['options'] = array_unique(array_merge($customOptions, $data['options']));
+$data['options'] = array_replace($customOptions, $data['options']);
```

Comment:
Yes array_merge() ok here, bug in array_unique().
array_replace and array_merge the same here, because not use number indexes here, so it not need mergings(combine number indexes), just replace associative keys.